packages/node_modules/@webex/internal-plugin-devices/src/devices.js

```diff
@@ -0,0 +1,753 @@
+// Internal dependencies.
+import {WebexPlugin} from '@webex/webex-core';
+import {safeSetTimeout} from '@webex/common-timers';
+
+import FeaturesModel from './features-model';
+
+/**
+ * Devices plugin.
+ *
+ * @description
+ * This plugin is used as a prototyped replacement plugin for the previous
+ * device plugin `internal-plugin-wdm`. After this plugin is fully implemented,
+ * other plugins will likely need to be modified to consume this one over
+ * `internal-plugin-wdm` [ **device** ].
+ */
+const Devices = WebexPlugin.extend({
+
+  // Ampersand property members.
+
+  namespace: 'Devices',
+
+  children: {
+    /**
+     * The class object that contains all of the feature collections.
+     *
+     * @type {FeaturesModel}
+     */
+    features: FeaturesModel
+  },
+
+  props: {
```

Comment:
Is there any way we can make this `props` value dynamic? Any time WDM adds a value, an engineer needs to update the SDK to add that one value.

Reply:
This can be done using `extraProperties: 'allow'`, will implement this to allow for better flexibility.